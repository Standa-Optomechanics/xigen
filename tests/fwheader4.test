#!/bin/sh
file_input="$srcdir/header4.xi"
. $srcdir/defs

cat < $file_expected > $file_ok

expect_out_fwheader
expect_consistent
