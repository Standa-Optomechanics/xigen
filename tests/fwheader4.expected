
#define PROTOCOL_VERSION 1
#define PROTOCOL_VERSION_Q "1"


// Command codes

#define SFO1_CMD	0x316f6673
#define GFO1_CMD	0x316f6667
#define SFBR_CMD	0x72626673
#define GFBR_CMD	0x72626667
#define RBBZ_CMD	0x7a626272
#define RQQX_CMD	0x78717172
#define SFO2_CMD	0x326f6673
#define GFO2_CMD	0x326f6667


typedef struct sfo1_cmd_str {
	uint32_t	speed;
} sfo1_cmd_str; 

typedef struct sfbr_cmd_str {
	int32_t	position;
} sfbr_cmd_str; 

typedef struct rbbz_cmd_str {
	int32_t	velocity;
} rbbz_cmd_str; 

typedef struct rqqx_cmd_str {
	int32_t	velocity;
} rqqx_cmd_str; 

typedef struct sfo2_cmd_str {
	uint32_t	speed;
} sfo2_cmd_str; 
